systemtest/src/main/java/io/strimzi/systemtest/utils/FileUtils.java

```diff
@@ -30,132 +30,73 @@ private FileUtils() { }
 
     @SuppressFBWarnings("RV_RETURN_VALUE_IGNORED_BAD_PRACTICE")
```

Comment:
Is this still needed?

Reply:
I have changed that now i will use return value from file.mkdirs() so it is not longer needed.